src/assets/custom/js/websiteNavigation.test.js

```diff
@@ -206,10 +233,12 @@ function createMockMenuToggle() {
 }
 
 function createMockMenu() {
+  const header = getMockHeader();
   const menu = window.document.createElement("div");
   menu.id = MENU_ID;
 
-  window.document.body.appendChild(menu);
+  // window.document.body.appendChild(menu);
```

Comment:
Should this be deleted?

Reply:
Good spot! https://github.com/codurance/site/pull/1933/commits/c96f603e23db76226c321fa87032f41538b0218f